code/Test_definitions/carrier-billing-retrievePayments.feature

```diff
@@ -0,0 +1,322 @@
+Feature: CAMARA Carrier Billing API, v0.3 - Operation retrievePayments
```

Comment:
In order to avoid confusion with next one what about retrievePaymentList

Reply:
I would prefer keeping in this release retrievePayment and retrievePayments, just to be aligned with the release history

We can talk within the WG to rename operationIds to retrievePaymentById and retrievePaymentList in the next Release track

Reply:
Got your rationale !
No problem for me.